Core/GDCore/Events/Builtin/CommentEvent.cpp

```diff
@@ -12,6 +12,29 @@ using namespace std;
 
 namespace gd {
 
+vector<const gd::String*> CommentEvent::GetAllSearchableStrings( BaseEvent &event)
+    const {
+  vector<const gd::String*> allSearchableStrings;

```

Comment:
vector<gd::String> allSearchableStrings;
```suggestion
  vector<const gd::String*> allSearchableStrings;
```